plugins/woocommerce/src/Internal/DataStores/Orders/FeatureController.php

```diff
@@ -0,0 +1,324 @@
+<?php
```

Comment:
I feel we should change filename to something more specific here as well.

Reply:
Maybe, but what name could we use? Something like `CustomOrdersTableController` perhaps?

Reply:
`CustomOrdersTableController` sounds good to me